packages/block-library/src/post-author/index.php

```diff
@@ -15,10 +15,14 @@
  */
 function render_block_core_post_author( $attributes, $content, $block ) {
 	if ( ! isset( $block->context['postId'] ) ) {
-		return '';
+		global $wp_query;
+		if ( $wp_query && isset( $wp_query->query_vars ) && is_array( $wp_query->query_vars ) ) {
+			$author_id = $wp_query->query_vars['author'];
+		}
```

Comment:
```suggestion
		$author_id = get_query_var( 'author' );
```

Suggestion (non-blocking): We could use [`get_query_var`](https://developer.wordpress.org/reference/functions/get_query_var/) here for simplicity.